.github/workflows/ci.yml

```
@@ -43,6 +43,11 @@ jobs:
         run: npm run build
       - name: Lint
         run: npm run is-it-pretty
+      - name: Unset git config safe.directory globally
+        run: git config --global --unset-all safe.directory || true
+      - name: Unset git config safe.directory for linux
+        if: runner.os == 'Linux'
+        run: sudo git config --system --unset-all safe.directory || true
```

It's unfortunate that they put the safe.directory entry in the system config on the Linux images. I think we can still achieve a unified solution to the problem though by leveraging the fact that `safe.directory` is a multi-value config key. See [git-config](https://git-scm.com/docs/git-config#:~:text=This%20is%20a%20multi%2Dvalued%20setting%2C%20i.e.%20you%20can%20add%20more%20than%20one%20directory%20via%20git%20config%20%2D%2Dadd.%20To%20reset%20the%20list%20of%20safe%20directories%20(e.g.%20to%20override%20any%20such%20directories%20specified%20in%20the%20system%20config)%2C%20add%20a%20safe.directory%20entry%20with%20an%20empty%20value.)

> This is a multi-valued setting, i.e. you can add more than one directory via git config --add. To reset the list of safe directories (e.g. to override any such directories specified in the system config), add a safe.directory entry with an empty value.

```suggestion
      - name: Reset safe.directory
        run: git config --global --add safe.directory ""
        shell: bash
```

The `shell: bash` part is needed to not have to do special escaping for empty arguments just for Windows' Powershell.